src/test/ui/mir/issue-80742.stderr

```diff
@@ -0,0 +1,42 @@
+error[E0599]: no function or associated item named `new` found for struct `Inline<dyn Debug>` in the current scope
+  --> $DIR/issue-80742.rs:31:36
+   |
+LL | / struct Inline<T>
+LL | | where
+LL | |     [u8; size_of::<T>() + 1]: ,
+LL | | {
+LL | |     _phantom: PhantomData<T>,
+LL | |     buf: [u8; size_of::<T>() + 1],
+LL | | }
+   | |_- function or associated item `new` not found for this
+...
+LL |       let dst = Inline::<dyn Debug>::new(0);
+   |                                      ^^^ function or associated item not found in `Inline<dyn Debug>`
+   | 
+  ::: $SRC_DIR/core/src/fmt/mod.rs:LL:COL
+   |
+LL |   pub trait Debug {
+   |   --------------- doesn't satisfy `dyn Debug: Sized`
+   |
+   = note: the method `new` exists but the following trait bounds were not satisfied:
+           `dyn Debug: Sized`
+
+error[E0277]: the size for values of type `dyn Debug` cannot be known at compilation time
+  --> $DIR/issue-80742.rs:31:15
+   |
+LL | struct Inline<T>
+   |               - required by this bound in `Inline`
+...
+LL |     let dst = Inline::<dyn Debug>::new(0);
+   |               ^^^^^^^^^^^^^^^^^^^^^^^^ doesn't have a size known at compile-time
+   |
+   = help: the trait `Sized` is not implemented for `dyn Debug`
+help: consider relaxing the implicit `Sized` restriction
+   |
+LL | struct Inline<T: ?Sized>
+   |                ^^^^^^^^
+
+error: aborting due to 2 previous errors
+
+Some errors have detailed explanations: E0277, E0599.
+For more information about an error, try `rustc --explain E0277`.
```

Comment:
@oli-obk I don't care about this output too much, I just don't want this program to break the compiler. Is there a way to tell "just run the test and expect it to fail (but not with ICE!)"?

Reply:
hmm... not really, this is how our UI test suite works. You can just ignore this file, it will keep getting autogenerated with `--bless`, and it doesn't actually matter what it says, all that matters is that the test keeps failing, which you checked with `//check-fail`

Reply:
> Is there a way to tell "just run the test and expect it to fail (but not with ICE!)"?

This is the default behaviour. The exit code for an ICE is different and checked by the compiletest tool that runs the tests.

Reply:
I mean I don't want to check in a .stderr file because I don't care about the actual error message here. I just want to test that the compiler does not crash when compiling this program.

Reply:
Ignoring stderr is also possible (dont-check-compiler-stderr or something like that), but would be rather unusual for a regression test to do so. I would recommend leaving this as is.